src/gui/src/SampleEditor/MainSampleWaveDisplay.cpp

```diff
@@ -167,8 +167,8 @@ void MainSampleWaveDisplay::updateDisplay( const QString& filename )
 
 		float fGain = height() / 4.0 * 1.0;
 
-		float *pSampleDatal = pNewSample->get_data_l();
-		float *pSampleDatar = pNewSample->get_data_r();
+		auto pSampleDatal = pNewSample->get_data_l();
```

Comment:
Is this intentional? There are also some other places where float * is replaced by auto. I was unsure if this might be an effect from auto-replacing text.

Reply:
Yeah. Somewhat. I toyed with having the sample data being represented as `std::vector` instead of a raw float arrays. I thought it would might be handy to be able to use the algorithms of the standard library on it and also to not have any raw audio leaked at some point in the future.

But it turned out copying takes about 4x as long for mono and 1.5x for stereo files. That was more than I expected. So I dropped these changes but left all the `auto` in place cause they don't hurt and it would make revisiting those changes faster.